labgrid/factory.py

```diff
@@ -147,6 +152,18 @@ def make_target(self, name, config, *, env=None):
             self.make_driver(target, driver, name, args)
         return target
 
+    def registered_class_from_string(self, string: str):
```

Comment:
The name is wrong (and too long to be convinient). Currently, this function will find even unregistered parent classes.